control-plane/csi-driver/controller/src/pvwatcher.rs

```diff
@@ -0,0 +1,145 @@
+use anyhow::Result;
+use futures::TryStreamExt;
+use k8s_openapi::api::core::v1::PersistentVolume;
+
+use kube::{
+    api::{Api, ListParams},
+    runtime::{watcher, WatchStreamExt},
+    Client,
+};
+use tracing::{debug, error, info};
+use uuid::Uuid;
+
+use crate::IoEngineApiClient;
+
+const MAYASTOR_PLUGIN: &str = "io.openebs.csi-mayastor";
+
+pub struct PvGarbageCollector {}
+
+impl PvGarbageCollector {
+    pub async fn start_watcher() -> Result<(), watcher::Error> {
+        info!("Starting PV Garbage Collector");
+        debug!("Handling if any missed events");
```

Comment:
why do we need this log?

Reply:
Line 21 or 22. Line 21 will indicate in logs that PV Garbage collector is started. Line 22 i will remove

Reply:
I have moved line 22 logger to the funcation that handles missed events (Made it info! as well). This runs at startup only btw.